homework-classes/Contributor.js

```diff
@@ -8,12 +8,32 @@ class Contributor {
     this.contributor = contributor;
   }
 
-  /**
-   * Render the contributor info to the DOM.
-   * @param {HTMLElement} container The container element in which to render the contributor.
-   */
-  render(container) {
-    // TODO: replace the next line with your code.
-    Util.createAndAppend('pre', container, { text: JSON.stringify(this.contributor, null, 2) });
+  render() {
```

Comment:
> In a previous version, I think you had a separate method to create a card which you called from this `render()` method. That is perfectly fine, and in fact, I would recommend it. Functions/methods should be small.
> 
You are right, in this homework they are also same, because i use creating cards function in render(), the rest are in the App.js selectRepository() function.